src/tests/validity/glob_absolute_missing_file.ab

```diff
@@ -0,0 +1,32 @@
+import * from "std/text"
+
+// Output
+// ===
```

Comment:
These separators are necessary because if the script creates no output (as in this case) the unit test actually behaves as if we had written:
```
// Output
// Succeded
```